be/src/olap/task/engine_clone_task.cpp

```diff
@@ -98,6 +98,12 @@ Status EngineCloneTask::_do_clone() {
         if (missed_versions.empty()) {
             LOG(INFO) << "missed version size = 0, skip clone and return success. tablet_id="
                       << _clone_req.tablet_id << " req replica=" << _clone_req.replica_id;
+            // update replica id to meet cooldown replica
```

Comment:
Seems only `if (_clone_req.replica_id != tablet->replica_id())` should reset replica id.

Reply:
I thought each clone task had one new replica id. It seems VERSION_INCOMPLETE task would has the same replica id. I'll go fix it.